### DISCUSSION TITLE
[feature request] Support `target="_blank"` in GitHub Flavoured Markdown

### DISCUSSION THREAD

**Discussion by HudsonGraeme** [community/community#38634] [Repositories]
This is a feature request to add the ability for users to specify a `target` attribute on links made within GFM.

[Many users in this issue](https://github.com/mojombo/github-flavored-markdown/issues/28) are requesting that GFM support anchor link targets to make it easier to open links in new tabs without needing to <kbd>command</kbd>+click every link. It'd be great to have a collaborator look into this issue as it's been open for 10 years now in the GFM repo and still gets replies. (Unfortunately, you'll have to middle or <kbd>command</kbd>+click that link above to open it in a new tab since GFM lacks target support).

**Comment by Kochise** [comment]
For reference : https://github.com/mojombo/github-flavored-markdown/issues/28 (opened Nov 23, 2012 = 10+ years)

Quite strangely, `target="_blank"` has **NOT** been wiped from https://github.com/mojombo/github-flavored-markdown/issues/28#issuecomment-242261616

So it works, it's just a matter of "not filtering it out".

**Comment by DSDmark** [comment]
Hi @HudsonGraeme,

add **[feature request]** in your title, it will catch attention of community.

**Comment by vishalps2606** [comment]
[feature request] 

Please Github team, look into this matter. Developers are asking for this feature for more than 10 years now and still no response.
[link to request](https://github.com/mojombo/github-flavored-markdown/issues/28)

**Comment by nexushoratio** [comment]
-1 for this feature.

As an end user (person using the browser), *I* want to be the one in control of whether or not to open a page in a new tab or not.

I don't know about other browsers, but for me on Firefox on Linux, it is a simple middle click to open in a new tab.

**Reply by kevinweejh** [reply]
The balanced approach: 
- Allow the use of the target attribute in the GFM spec, while 
- Insert warnings in bold to remind users to be mindful and use it sparingly. 

Allow me to explain: 
I am of the mind that Markdown users should have access to the target attribute, but they should only be using it where appropriate, such as linking to external sites. This helps them better ensure task continuity for the user, while improving user retention. However, these links should have something like "(Opens in new tab)" appended at every instance, to give explicit notification - which is especially important for screen reader users. Where we should trust most end users to be capable of using the necessary keyboard shortcut to open links in a new tab on their own, I believe we can and should also trust developers to judiciously use such a feature where appropriate.

**Reply by spookyahell** [reply]
You want control over your links?
You already have that...
You can just drag and drop a link on the tab.
That'll always open it in the same window.

The default is to let project owner decide how the documentation is handled.
And the gold web standard is to open external links in a new tab as not to overwrite the currrent one.
Could just move the whole README stuff to gh-pages.

**Comment by sokai** [comment]
Hey!
I _suggest_ to change this request to a **support for [all `target` options](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/a#target)**.
I _need_ to use `target="_self"` (and now/ATM _have to_ use HTML links in my markdown).
Thanks and KR!
